tests/integration/test_definitions.py

```diff
@@ -127,7 +127,43 @@ def define_tests():
                 FileHasNoRegex("./merlin_server/redis.users", "new_user"),
             ],
             "local",
-            CLEAN_MERLIN_SERVER,
+            "rm -rf merlin_server/ hello*/",
+        ),
+    }
+    server_integrated_tests = {
+        "celery task id": (
+            """merlin server init:
```

Comment:
I think your tests are stuck in limbo because maybe this should be semi-colon?

Reply:
I killed the tests after 1 hour

Reply:
I think you are right thanks for catching that. Let me fix it and see if I still stalls.